tests/urls.py

```diff
@@ -1,15 +1,20 @@
 # -*- coding: utf-8 -*-
 
-from django.conf.urls import include, url
+from django.urls import path
+from django.conf.urls import include
 
 from rest_framework import routers
 
 from tests.mockapp.views import SearchPersonFacetViewSet, SearchPersonMLTViewSet
 
 router = routers.DefaultRouter()
-router.register("search-person-facet", viewset=SearchPersonFacetViewSet, basename="search-person-facet")
-router.register("search-person-mlt", viewset=SearchPersonMLTViewSet, basename="search-person-mlt")
+router.register(
+    "search-person-facet",
+    viewset=SearchPersonFacetViewSet,
+    basename="search-person-facet",
+)
+router.register(
+    "search-person-mlt", viewset=SearchPersonMLTViewSet, basename="search-person-mlt"
+)
```

Comment:
isn't 120 a suitable max line length today? sticking to 80 is a bit of an old thing, no?